app/views/static/notification_drawer/notification-drawer.html.haml

```diff
@@ -7,26 +7,26 @@
     %h3.text-center
       {{drawerTitle}}
 
-  .drawer-pf-title{'ng-if'      => "titleInclude",
-                   'ng-include' => "titleInclude"}
-
   %div{'pf-fixed-accordion' => true,
        'scroll-selector'    => ".panel-body"}
     .panel-group
       .panel.panel-default{'ng-repeat' => "notificationGroup in notificationGroups track by $index"}
         .panel-heading
           %h4.panel-title
             %a{'ng-click'   => "toggleCollapse(notificationGroup)",
-               'ng-class'   => "{collapsed: !notificationGroup.open}",
-               'ng-include' => "headingInclude"}
-          %span.panel-counter{'ng-include' => "subheadingInclude"}
+               'ng-class'   => "{collapsed: !notificationGroup.open}"}
+              %span.heading-class
+                {{notificationGroup.heading}}
+          %span.panel-counter
+            %span.subheading-class
+              {{customScope.unreadCountText(notificationGroup.unreadCount)}}
 
         .panel-collapse.collapse{'ng-class' => "{in: notificationGroup.open}",
                                  'ng-if'    => "notificationGroup.open"}
           .panel-body
             .drawer-pf-notification{'ng-class'   => "{unread: notification.unread, 'expanded-notification': drawerExpanded}",
-                                    'ng-repeat'  => "notification in notificationGroup.notifications | limitTo:limit.notifications:(notificationGroup.notifications.length - limit.notifications)",
-                                    'ng-include' => "notificationBodyInclude"}
+                                    'ng-repeat'  => "notification in notificationGroup.notifications | limitTo:limit.notifications:(notificationGroup.notifications.length - limit.notifications)"}
+              = render :partial => 'static/notification_drawer/notification-body'
```

Comment:
Any reason why this needs to be in a partial?

Reply:
No reason at all, just didn't want to make the file unreadable large.

Reply:
Fair enough, you don't have to!